Source/WebCore/platform/mediastream/mac/CGDisplayStreamCaptureSource.cpp

```diff
@@ -43,6 +43,11 @@
 
 namespace WebCore {
 
+CGDisplayStreamCaptureSource::CGDisplayStreamCaptureSource()
```

Comment:
Should we do this for both screen capture classes?

Reply:
The other classes are run in GPUProcess and I did not see it being app napped.
That said, you are right, let's move this UserActivity to DisplayCaptureSourceCocoa which is the one handling the timer.